executors/sshexec/volume.go

```diff
@@ -155,6 +158,33 @@ func (s *SshExecutor) VolumeDestroyCheck(host, volume string) error {
 	return nil
 }
 
+func (s *SshExecutor) createVolumeOptionsCommand(volume *executors.VolumeRequest) []string {
+
+	godbc.Require(len(volume.Options) > 0)
+
+	commands := []string{}
+	var cmd string
+
+	// Go through all the Options and create volume set command
+	for _, volOption := range volume.Options {
+		if volOption != "" {
+			switch volOption {
+			case "encryption":
+				cmd = fmt.Sprintf("gluster --mode=script volume set %v server.ssl on", volume.Name)
```

Comment:
@obnoxxx can we discuss about the options later and do more enhancements with a follow up patch ?